lib/IRGen/GenClangDecl.cpp

```diff
@@ -86,6 +96,9 @@ void IRGenModule::emitClangDecl(const clang::Decl *decl) {
       if (DC->isFileContext())
         break;
       D = cast<const clang::Decl>(DC);
+      if (DC->isRecord()) {
```

Comment:
Nit: can you move this above line 98 so it's with the similar checks?

Reply:
Done.